src/testing/test-helpers.ts

```diff
@@ -0,0 +1,22 @@
+import { AxePuppeteer } from '@axe-core/puppeteer';
+
+const ignoredDefaults = ['document-title', 'html-has-lang', 'page-has-heading-one', 'landmark-one-main', 'region'];
+
+const buildAxeErrorOutput = violations =>
+  violations
+    .map(
+      v =>
+        `[${v.impact}] ${v.help}
+Id: ${v.id}
+See ${v.helpUrl}
+${v.nodes.reduce((str, { html, target }) => [str, html, ...target].join('\n\n'), '')}`,
+    )
```

Comment:
This output formatting was taken from the existing `axeCheck` function:

https://github.com/department-of-veterans-affairs/component-library/blob/a063396523f9484e730a173f7b520064a3480db2/src/helpers/test-helpers.js#L48-L57